components.md

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
12. This is a component page I've created to test the sidebar layout as well as theming of other USWDS components, for example: alerts.

Reply:
We've reached the end 🥳

To summarize we've:
1. Installed USWDS
1. Imported it into our `vendor/` directory
1. Created our own `styles.scss` and our own overrides partial with branding.
1. Added some global USWDS components
1. Created some basic templates to view our new styles